src/stravalib/strava_model.py

```diff
@@ -3,10 +3,9 @@
 
 from __future__ import annotations
 
-from datetime import datetime
 from typing import Literal, Optional
 
-from pydantic import BaseModel, Field
+from pydantic import AwareDatetime, BaseModel, Field, RootModel
```

Comment:
I'm mostly curious about what this `AwareDatetime` will do.

Reply:
@jsamoocha i know this is a very old pr now but when i run the model locally, i can't get it to actually use these new pydantic objects. Did you run the code locally and it added AwareDatetime? it's reverting your changes every time i run it locally. 

https://github.com/stravalib/strava_swagger2pydantic/pull/14/files